WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -32,7 +37,7 @@ public static EditPostPreviewFragment newInstance(SiteModel site, PostModel post
         EditPostPreviewFragment fragment = new EditPostPreviewFragment();
         Bundle bundle = new Bundle();
         bundle.putSerializable(WordPress.SITE, site);
-        bundle.putSerializable(EditPostActivity.EXTRA_POST, post);
+        bundle.putInt(EditPostActivity.EXTRA_POST_LOCAL_ID, post.getId());
```

Comment:
> because we want to avoid passing ids
Can you elaborate on this? I am just curious about it. Because, we can avoid the confusion about the remote & local ids issue by just explicitly naming them like `EXTRA_POST_LOCAL_ID` or `EXTRA_POST_REMOTE_ID`. It's no more confusing than having the remote and local id versions of getters in our stores.

In general I thought it was better to pass the ids, especially to avoid crashes like this. I can't see what we get out of passing the `serializable` except for maybe convenience.

Please note that I am not against it, I just want to better understand it.